deploy/docs/v2_migration_doc.md

```diff
@@ -82,3 +82,65 @@ the exact steps for migration.
 ## How to upgrade
 
 **Note: The below steps are using Helm 3. Helm 2 is not supported.**
+
+### 1. Upgrade to helm chart version `v1.3.2`
+
+Run the below command to fetch the latest helm chart:
+
+```bash
+helm repo update
+```
+
+For the users who are not already on `v1.3.2` of the helm chart, please upgrade
+to that version first by running the below command.
+
+```bash
+helm upgrade collection sumologic/sumologic --reuse-values --version=1.3.2
+```
+
+### 2. Upgrade Prometheus CRDs
+
+Due to changes in `kube-prometheus-stack` which this chart depends on, one will
+need to run the following commands in order to update Prometheus related CRDs.
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/release-0.43/example/prometheus-operator-crd/monitoring.coreos.com_probes.yaml
+kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/release-0.43/example/prometheus-operator-crd/monitoring.coreos.com_alertmanagers.yaml
+kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/release-0.43/example/prometheus-operator-crd/monitoring.coreos.com_alertmanagerconfigs.yaml
+kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/release-0.43/example/prometheus-operator-crd/monitoring.coreos.com_prometheuses.yaml
+kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/release-0.43/example/prometheus-operator-crd/monitoring.coreos.com_prometheusrules.yaml
+kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/release-0.43/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml
+kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/release-0.43/example/prometheus-operator-crd/monitoring.coreos.com_podmonitors.yaml
+kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/release-0.43/example/prometheus-operator-crd/monitoring.coreos.com_thanosrulers.yaml
+```
+
+### 3. Run upgrade script
+
+For Helm users, the only breaking changes are the renamed config parameters.
+For users who use a `values.yaml` file, we provide a script that users can run
+to convert their existing `values.yaml` file into one that is compatible with the major release.
+
+- Get the existing values for the helm chart and store it as `current_values.yaml`
+  with the below command:
+
+  ```bash
+  helm get values <RELEASE-NAME> > current_values.yaml
+  ```
+
+- Run `curl` the upgrade script as follows:
+
+  ```bash
+  curl -LJO https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/main/deploy/helm/sumologic/upgrade-2.0.0.sh
+  ```
+
+- Run the upgrade script on the above file with the below command.
+
+  ```bash
+  ./upgrade-2.0.0.sh current_values.yaml
```

Comment:
Maybe before this step we should add `chmod +x upgrade-2.0.0.sh`. It is quite obvious but usually this step appears in instructions, e.g. [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-binary-with-curl-on-linux) and [here](https://helm.sh/docs/intro/install/#from-script)

Reply:
I'm testing on vagrant VM created from `release-v1.3` branch, commit: 2fac768fb.
I see following error:
```
./upgrade-2.0.0.sh current_values.yaml
[INFO]    Mapping fluentd.image.repository into:
[INFO]    Mapping fluentd.image.tag into:
[INFO]    Mapping image.repository into:
[INFO]    - fluentd.image.repository
[INFO]    - sumologic.setup.job.image.repository
[INFO]    Mapping image.tag into:
[INFO]    - fluentd.image.tag
[INFO]    - sumologic.setup.job.image.tag

[INFO]    Migrating prometheus-config-reloader container to config-reloader in prometheusSpec
Error: unknown flag: --arrays
```

yq version: 
```
$ yq --version
yq version 3.3.0
```

Reply:
Good points

Reply:
You'll need `yq` at least at version `3.4.0` 

The following handles that at VM provision time but you can run it manually

https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/63c6cf3415ed5be396834d527f51612fa8382f66/vagrant/provision.sh#L53-L56

Reply:
I'm testing from `release-v1.3` branch,  https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/release-v1.3/vagrant/provision.sh#L56

May we mention required `yq` version in this instruction?

Reply:
Ok, apparently I missed and incorrectly changed the v2 update script

it should be fixed (i.e. user will get a notification about required `yq` version) via:

https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/1172/files#diff-e010a115e1f3b27c1977c58d40e6aad5860cb28d7beb501161fb7d4a9358e6cc